recipes/ldrb/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set version = "2021.0.1" %}
+
+package:
+  name: ldrb
+  version: {{ version }}
+
+source:
+  url: https://github.com/finsberg/ldrb/archive/v{{ version }}.tar.gz
+  sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+
+build:
+  number: 0
+  skip: True  # [win or py<36]
```

Is windows completely unsupported? Or are there missing dependencies?

---

This package depends on [FEniCS](https://anaconda.org/conda-forge/fenics) which does not support windows, so yes, windows is completely unsupported.

---

It seems like this package is only pure Python? In that case it could be made noarch Python, which would drastically cut down on the number of builds. The Windows build here would fail due to missing fenics, which is fine and we could ignore. If that's the case:
```suggestion
  noarch: python
```

---

> It seems like this package is only pure Python? In that case it could be made noarch Python, which would drastically cut down on the number of builds. The Windows build here would fail due to missing fenics, which is fine and we could ignore. If that's the case:

You are right that the package is pure python.